client/Dockerfile-dev

```diff
@@ -1,5 +1,3 @@
-ARG AWS_REGISTRY
-ARG VERSION
-FROM $AWS_REGISTRY/digideps/client:$VERSION
+FROM client:latest
```

Comment:
Where do we use this Dockerfile? Is it just deployed to the main dev environment or PRs as well? Might cause oddities if so.

Reply:
So this is part of the build to create the development environment image. The development environment image is largely the same as our normal image with a minor tweak so we build client and tag it with client:latest locally and we then build this image from that so it's not pulling from any repository or anything :)